Econify/graphql-rest-router
# [MM, NG] Typescript declarations && example application

Fix:
* npm vulnerabilities
* tests
* constructor / type declarations and exports

Add:
* Linting
* Test gates for commit
* Example application for testing prs in the future

Etc.:
* Bring dependencies into 2021
* Install no longer runs build
* Change directory structure to have a build folder

For example app:
* From root directory run `npm run live-test`.
* This will install all needed dependencies and boot up a web application
* This application will ensure all rest router code is still working.
* Todos for this app are in the nested read me.
